README.md

```diff
@@ -2,11 +2,11 @@
 # Snowflake
 
 Publisher: Splunk  
-Connector Version: 1.1.0  
+Connector Version: 1.1.1  
```

Comment:
This is totally fine, but you don't actually need to manually update the app version - it will get updated when the job runs to create the PR from next -> main.

Reply:
Ok, great to know, thanks